lib/themes/dosomething/paraneue_dosomething/js/utilities/Analytics.js

```diff
@@ -4,97 +4,52 @@
  * or form validation events.
  */
 
-import $ from 'jquery';
-import Validation from 'dosomething-validation';
-import Modal from 'dosomething-modal';
+const $ = require('jquery');
+const Analytics = require('@dosomething/analytics');
+const Validation = require('dosomething-validation');
+const Modal = require('dosomething-modal');
+
+function init() {
+  // Configure `data-ga-*` and `data-track-*` links.
+  Analytics.init('ga');
+  Analytics.init('track');
```

Comment:
Makes sense :) Can leave for now, and maybe do the latter (remove old `-ga-` attributes) in a future cleanup PR? Up to you, but not vital for merge 😄